src/Raven.Client/Documents/BulkInsert/BulkInsertWriterBase.cs

```diff
@@ -135,7 +136,10 @@ private async Task WriteToStreamAsync(Stream src, Stream dst, JsonOperationConte
             await dst.WriteAsync(buffer.Memory.Memory.Slice(0, bytesRead), _token).ConfigureAwait(false);
 
             if (forceDstFlush)
+            {
+                lastFlushToStream = DateTime.UtcNow;
```

Comment:
To be more accurate, it's not just about the last flush but about `lastForcedFlushToStream`, right?

Reply:
yes